Alignment/APEEstimation/macros/DrawIteration.C

```diff
@@ -665,8 +669,8 @@ void DrawIteration::drawFinals(const std::string& xOrY){
       std::vector<Input*>::const_iterator i_input;
       for(i_input = v_input_.begin(); i_input != v_input_.end(); ++i_input, ++iInput){
         sectorValues = &(*i_input)->sectorValues;
-	TH1* hist(0);
-	TString& legendEntry = (*i_input)->legendEntry;
+  TH1* hist(0);
```

Comment:
please use `nullptr` instead of `0`

Reply:
Consistently using nullptr in this file, now.